recipes/reflectcpp/all/conanfile.py

```diff
@@ -0,0 +1,148 @@
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/getml/reflect-cpp"
+    topics = (
+        "reflection",
+        "serialization",
+        "memory",
+        "cbor",
+        "flatbuffers",
+        "json",
+        "msgpack",
+        "toml",
+        "xml",
+        "yaml",
+    )
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cbor": [True, False],
+        "with_flatbuffers": [True, False],
+        "with_msgpack": [True, False],
+        "with_toml": [True, False],
+        "with_ubjson": [True, False],
+        "with_xml": [True, False],
+        "with_yaml": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cbor": False,
+        "with_flatbuffers": False,
+        "with_msgpack": False,
+        "with_toml": False,
+        "with_ubjson": False,
+        "with_xml": False,
+        "with_yaml": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def requirements(self):
+        self.requires("ctre/3.9.0", transitive_headers=True)
+        self.requires("yyjson/0.10.0", transitive_headers=True)
+        if self.options.with_cbor:
+            self.requires("tinycbor/0.6.0", transitive_headers=True)
+        if self.options.with_flatbuffers:
+            self.requires("flatbuffers/24.3.25", transitive_headers=True)
+        if self.options.with_msgpack:
+            self.requires("msgpack-c/6.0.0", transitive_headers=True)
+        if self.options.with_toml:
+            self.requires("tomlplusplus/3.4.0", transitive_headers=True)
+        if self.options.with_ubjson:
+            self.requires("jsoncons/0.176.0", transitive_headers=True)
+        if self.options.with_xml:
+            self.requires("pugixml/1.14", transitive_headers=True)
+        if self.options.with_yaml:
+            self.requires("yaml-cpp/0.8.0", transitive_headers=True)
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.23 <4]")
+
+    def validate(self):
+        if self.settings.get_safe("compiler.cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support.")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
```

Comment:
```suggestion
        get(self, **self.conan_data["sources"][self.version], strip_root=True)
        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"), "set(CMAKE_CXX_STANDARD 20", "")
```

Let Conan configure the C++ standard based on settings.compiler.cppstd. Otherwise, will use always C++20.